applications/IgaApplication/custom_python/iga_python_application.cpp

```diff
@@ -48,6 +48,11 @@ PYBIND11_MODULE(KratosIgaApplication, m)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RAYLEIGH_ALPHA)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RAYLEIGH_BETA)
 
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, POINT_LOAD)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, SURFACE_NORMAL)
```

Comment:
why not just NORMAL?

Reply:
We still should move Rayleigh variables to core

Reply:
Still not sure if we should do this
This is not sth every app could use, therefore I think it is fine if defined in the apps

Reply:
It is common across many apps, we are registering them again and again

Reply:
I agree on you Vicente. But, I think we already lost a discussion about this ;)

About the SURFACE_NORMAL -> it is more explenatory but I guess NORMAL could be enough...